dags/pensieve.py

```diff
@@ -0,0 +1,59 @@
+from airflow import DAG
+from datetime import timedelta, datetime
+from operators.gcp_container_operator import GKEPodOperator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+
+default_args = {
+    'owner': ['ascholtz@mozilla.com', 'sunahsuh@mozilla.com', 'tdsmith@mozilla.com'],
+    'depends_on_past': False,
+    'start_date': datetime(2020, 03, 9),
+    'email_on_failure': True,
+    'email_on_retry': True,
+    'retries': 2,
+    'retry_delay': timedelta(minutes=30),
+}
+
+with DAG('pensieve',
+         default_args=default_args,
+         schedule_interval='@daily') as dag:
+
+    # Built from repo https://github.com/mozilla/pensieve
+    pensieve_image='gcr.io/moz-fx-data-experiments/pensieve:latest'
+
+    pensieve = GKEPodOperator(
+        task_id="pensieve",
+        name='pensieve',
+        image=pensieve_image,
+        email=['ascholtz@mozilla.com', 'sunahsuh@mozilla.com', 'tdsmith@mozilla.com'],
+        dag=dag)
+
+    wait_for_clients_daily_export = ExternalTaskSensor(
+        task_id="wait_for_clients_daily_export",
+        external_dag_id="main_summary",
+        external_task_id="clients_daily_export",
+        dag=dag)
+
+    wait_for_main_summary_export = ExternalTaskSensor(
+        task_id="wait_for_main_summary_export",
+        external_dag_id="main_summary",
+        external_task_id="main_summary_export",
+        dag=dag)
+
+    wait_for_search_clients_daily = ExternalTaskSensor(
+        task_id="wait_for_search_clients_daily",
+        external_dag_id="main_summary",
+        external_task_id="search_clients_daily",
+        dag=dag,
+    )
+
+    wait_for_event_events = ExternalTaskSensor(
+        task_id="wait_for_event_events",
+        external_dag_id="main_summary",
+        external_task_id="event_events",
+        dag=dag,
+    )
+
+    pensieve.set_upstream(wait_for_clients_daily_export)
+    pensieve.set_upstream(wait_for_main_summary_export)
+    pensieve.set_upstream(wait_for_search_clients_daily)
+    pensieve.set_upstream(wait_for_event_events)
```

Comment:
I'm not completely sure if those are correct. I looked at the datasources that mozanalysis uses for its analysis: https://github.com/mozilla/mozanalysis/blob/561e1ad650d81a940562e7090b5ca5757f87c55d/src/mozanalysis/metrics/desktop.py

Reply:
At least for the metrics in that file it looks like we need clients daily, search clients daily, and events.